smart-metastore/src/main/java/org/smartdata/metastore/tables/AccessCountDao.java

```diff
@@ -49,7 +74,7 @@ public static String createTableSQL(String tableName) {
   public String aggregateSQLStatement(AccessCountTable destinationTable
       , List<AccessCountTable> tablesToAggregate) {
     StringBuilder statement = new StringBuilder();
-    statement.append("CREATE TABLE '" + destinationTable.getTableName() + "' as ");
+    statement.append("CREATE TABLE " + destinationTable.getTableName() + " as ");
```

Comment:
Quotes are suggested by mysql. But in current setting (druid with mysql), I found that quotes will caused wrong sql exception.  Please remove all single quotes.